english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md

```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```

Comment:
I add a link in that person's .md file so reader can choose "travel to Berlin" which link to my .md file